### PR TITLE
Translate Ruby Prize 2016 post (pt)

### PR DESCRIPTION
**Pull request by jcserracampos**

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
cc @alanwillms & @fpgentil

File: pt/2016-07-25-ruby-prize-2016.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+O Ruby Prize é dado para reconhecer os esforços de atividades memoráveis e
+conquistas na comunidade do Ruby. O prêmio será entregue pelo comitê
+executivo composto por três organizações, a Ruby Association, Nihon Ruby no Kai
+e Matsue City.
+
+O ganhador e indicado finalista (1-2 pessoas) receberão uma premiação na
+RubyWorld Conference 2016, que ocorrerá em Matsue, Japão em 3 e 4 de novembro.
+
+Em adição, o ganhador do Ruby Prize também ganhará 1 milhão de ienes
+(aproximadamento 9.683 dólares no câmbio de 12 de julho de 2016)
```
Comment (fpgentil) [inline pt/2016-07-25-ruby-prize-2016.md]: `aproximadamente`

**Comment by stomar** [comment]
@jcserracampos Could you apply those suggestions and amend your commit?

**Comment by jcserracampos** [comment]
@stomar Yes, I'll fix today.

**Comment by jcserracampos** [comment]
@stomar sorry. I'm little late, but I fixed and amended

**Pull request merged**